src/config.coffee

```diff
@@ -413,7 +413,7 @@ class Config
   # * `keyPath` (optional) {String} name of the key to observe. Must be
   #   specified if `scopeDescriptor` is specified.
   # * `optional` (optional) {Object}
```

Comment:
I noticed that too, but it was unrelated to the `scope` change I was making here. I can add that too though if you like.